static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.js

```diff
@@ -1,18 +1,23 @@
 // @flow strict
 import config from 'lib/config';
 
-const onLoad = () => {
-    // Insert Twitter Pixel ID and Standard Event data below
-    if (window.twq) {
-        window.twq('init', 'nyl43');
-        window.twq('track', 'PageView');
-    }
+const insertSnippet = () => {
+    // Twitter universal website tag code -->
+    /*eslint-disable */
+    // $FlowFixMe
+        !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe(...arguments):s.queue.push(arguments);
```

Comment:
is this documented somewhere? a link to it in a comment might be useful if so

Reply:
exact generated code snippet is behind guardian's account login, but I have included a how-to doc